electionleaflets/apps/leaflets/tests/data.py

```diff
@@ -1,2 +1,291 @@
 MAPIT_POSTCODE_RETURN = """
```

Comment:
I think this is already not used anywhere, even before we get rid of `geocode()`